src/components/Comment/Comment/Comment.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactNode } from 'react';
+import { CommentData } from 'src/types/domain';
+import {
+  formatLongPublishDateString,
+  formatPublishDateString,
+} from 'src/utils/formatting';
+import {
+  CommentAuthor,
+  CommentContent,
+  CommentCreated,
+  CommentWrapper,
+  CommentAction,
+} from './styled';
+
+type CommentProps = {
+  // Additional classes
+  className?: string;
+
+  // Primary content
+  children?: ReactNode;
+
+  // Collapsed or hidden from view
+  collapsed?: boolean;
+
+  commentData: CommentData;
+
+  setReplyToCommentId: Function;
+
+  setReplyToCommentAuthor: Function;
+
+  isTopLevel?: boolean;
+
+  setFocus: () => void;
+};
+
+function Comment({
+  commentData,
+  children,
+  setReplyToCommentId,
+  setReplyToCommentAuthor,
+  isTopLevel = true,
+  setFocus,
+}: CommentProps) {
+  const handleClickReply = function () {
+    setReplyToCommentAuthor(commentData.byPerson.firstName);
+    setReplyToCommentId(commentData.id);
+    setFocus();
```

Comment:
Hm, I think you can refactor this into a single `onReply` callback instead? Then the parent can handle it however it likes to